From 31543921ba387f8a7455dfc110c742b2c883a983 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Tue, 28 Jan 2025 18:01:16 +0100
Subject: [PATCH 1566/1695] drivers: stm32mp_pmic2: drop useless include file

There is no use of the include file stm32_exti.h
Drop it.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git ef8299f6aee956ee542b1ed148af020430f1c429 drivers: stm32mp_pmic2: drop useless include file]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I8808d129cdb4fceffea646307472f80913d95766
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/434913
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/drivers/stm32mp_pmic2.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/drivers/stm32mp_pmic2.c b/core/drivers/stm32mp_pmic2.c
index fcd4d085f..73143afba 100644
--- a/core/drivers/stm32mp_pmic2.c
+++ b/core/drivers/stm32mp_pmic2.c
@@ -5,7 +5,6 @@
 
 #include <config.h>
 #include <drivers/regulator.h>
-#include <drivers/stm32_exti.h>
 #include <drivers/stm32_i2c.h>
 #if defined(CFG_STM32MP25) || defined(CFG_STM32MP23) || defined(CFG_STM32MP21)
 #include <drivers/stm32mp25_pwr.h>
-- 
2.47.2

